modules/apps/data-engine/data-engine-test/src/testFunctional/tests/macros/DataEngineUpgrade.macro

```diff
@@ -73,7 +73,9 @@ definition {
 	macro validateNWebContent {
 		WebContentNavigator.openWebContentAdmin(siteURLKey = "guest");
 
-		for (var wcTitle : list "My Web Content based on My Structure with Fieldsets level 2,My Web Content based on My Structure with all fields,My Web Content based on My Structure with nested fields,My Web Content based on My Structure with repeated fields") {
```

Comment:
Hey @ViniLopes20 , please notice that this commit message is discouraged. I've updated and you can see on my forwarded PR. Thanks